packages/data/addon/adapters/facts/bard.ts

```diff
@@ -266,6 +266,24 @@ export default class BardFactsAdapter extends EmberObject implements NaviFactAda
     return undefined;
   }
 
+  /**
+   * Builds rollup query params
+   * @param {RequestV1} request
+   * @return {String}
+   */
+  _buildRollupParam(request: RequestV2): string {
+    const columns = request.columns;
+    const rollupColumnCids = request.rollup?.columns || [];
+    return [
+      ...columns.reduce((colSet, requestColumn) => {
+        if (rollupColumnCids.includes(requestColumn.cid ?? '')) {
```

Comment:
can `rollupColumnCids` actually include `''` or is it just to return false?

Reply:
just to return false.